demos/projects/ESPRESSIF/az-ca-recovery/README.md

```diff
@@ -117,9 +117,11 @@ az account list-locations | ConvertFrom-Json | format-table -Property name
 # Create a resource group
 az group create --name '<name>' --location '<location>'
 
-# Deploy the resources
-# For the `resourcePrefix`, choose a unique, short name with only letter characters which
-# will be prepended to all of the deployed resources.
+# Deploy the resources. This may take a couple minutes.
+#  - `resourcePrefix`: choose a unique, short name with only lower-case letters which
+#                      will be prepended to all of the deployed resources.
```

Comment:
Nit "will be prepended" - consider "the script will prepend" to make it clear that the user has no further intervention to make after deciding this.